README.md

```diff
@@ -1,46 +1,71 @@
 # DiffOpt.jl
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://jump.dev/DiffOpt.jl/stable)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://jump.dev/DiffOpt.jl/dev)
 [![Build Status](https://github.com/jump-dev/DiffOpt.jl/workflows/CI/badge.svg?branch=master)](https://github.com/jump-dev/DiffOpt.jl/actions?query=workflow%3ACI)
 [![Coverage](https://codecov.io/gh/jump-dev/DiffOpt.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/jump-dev/DiffOpt.jl)
+[![Join the slack chat at https://app.slack.com/client/T68168MUP/C01409JFBMY](https://img.shields.io/badge/chat-on%20slack-blue.svg)](https://app.slack.com/client/T68168MUP/C01409JFBMY)
```

Comment:
Not sure we should point to slack, maybe directly to julialang.org/community/ for the different discussion channels

Reply:
- we should point to that link, although not via a badge
- is it spamming we're trying to avoid by putting the slack channel?  i saw packages had a gitter channel like a badge